docs/index.html

```diff
@@ -55,14 +55,8 @@ <h3>Stackdriver Sandbox is an open source tool* that helps practitioners to lear
       <h2>Get started now</h2>
       <div class="steps">
         <div class="step step-1">
-          <label>Step 1</label>
-          <a class="console-btn" href="https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/stackdriver-sandbox.git&cloudshell_git_branch=v0.1.14&cloudshell_working_dir=terraform&cloudshell_image=gcr.io/cloudshell-images/cloudshell" target="_blank" onclick="metrics.recordCustomTime('stackdriver-demo-open-in-cloud-shell')">Open in Google Cloud Shell</a>
-        </div>
-	<div class="step step-2">
-          <label>Step 2</label>
-          <p>In the Cloud Shell command prompt, type this command to install the Sandbox</p>
-          <code onclick="navigator.clipboard.writeText('./install.sh'); metrics.recordCustomTime('stackdriver-demo-copy-code-for-cloud-shell')">./install.sh</code>
-          <div class="line"></div>
+          <!--label>Step 1</label-->
+          <a class="console-btn" href="https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/stackdriver-sandbox.git&cloudshell_git_branch=v0.1.14&cloudshell_working_dir=terraform&cloudshell_image=gcr.io/natural-night-283917/custom-sandbox-image-1" target="_blank" onclick="metrics.recordCustomTime('stackdriver-demo-open-in-cloud-shell')">Open in Google Cloud Shell</a>
```

Comment:
@eaberman, we need to store the dockerfile that controls the content of this image - in our repository. We also need to update the CI to test that building the docker image using the dockerfile succeeds.

Reply:
Okay--currently I'm storing the image in a GCR repository. Do you mean that it would be better to store the Dockerfile in the stackdriver-sandbox project, or still in a GCR repository, but one that is intended solely for our project?

Reply:
@eaberman There are two important artifacts we are dealing with here:
- the Dockerfile/source code
  - important for making changes to our image later
  - should live in a git repo for collaboration and to track changes over time
- the compiled container image
  - this is needed to actually run our code in Docker or Kubernetes
  - think of it like a static binary that can be shared, but not as easily modified
  - should live in a registry like GCR

Reply:
+1 to what Dan said.
Container image for custom cloud shell image should live in project **stackdriver-sandbox-230822**
Dockerfile for the custom cloud shell image should live in this repo. You can create a cloudshell folder under the root and host it there. Dockerfile content should be part of this PR.

Reply:
Okay, I added some preliminary Dockerfile content, which I am working on testing.

Do you have any advice on where to start with the CI tests for the Dockerfile?